### PR TITLE
add UpsertTeamRequest to v1structs

### PR DESCRIPTION
**Pull request by paulfurley**

### CODE REVIEW

File: v1structs/v1structs.go [hunk 1]
```diff
@@ -84,6 +84,19 @@ type SecretMetadata struct {
 	SecretUUID string `json:"secretUuid"`
 }
 
+// UpsertTeamRequest is the JSON structure containing a signed team roster.
+type UpsertTeamRequest struct {
+	// TeamRoster describes the members and configuration of a team.
+	// See github.com/fluidkeys/fluidkeys/teamroster
```
Comment (idrysdale) [inline v1structs/v1structs.go]: This results in a 404. Should you include `/blob/master`?

File: v1structs/v1structs.go [hunk 2]
```diff
@@ -84,6 +84,19 @@ type SecretMetadata struct {
...
+	TeamRoster string `json:"teamRoster"`
+
+	// The ASCII-armored detached signature of the team roster.
+	// In GnuPG this would be generated by typing:
+	//
+	// > gpg --armor --output roster.toml.sig --detach-sig roster.toml
```
Comment (idrysdale) [inline v1structs/v1structs.go]: I tested this and it worked! 👍

File: v1structs/v1structs.go (hunk 1 above)
Comment (paulfurley) [reply v1structs/v1structs.go]: This is only the struct, I haven't made the endpoint yet !!!

Comment (idrysdale) [reply v1structs/v1structs.go]: My mistake, I interpretted this as a web url.

**Pull request merged**